ForwardedHeaderMiddleware/AspNetCore.ForwardedHeaderMiddleware/ForwardedHttpMiddleware.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace AspNetCore.ForwardedHttpMiddleware
+{
+    public class ForwardedHttpMiddleware
```

Comment:
Sorry, I'd meant to suggest ForwardedHttpMiddleware as the class name, with AspNetCore.ForwardedHttp as the assembly and namespace.